src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/integration/SqsIntegrationTestBase.kt

```diff
@@ -86,6 +86,8 @@ class SqsIntegrationTestBase : TestBase() {
     reset(telemetryClient)
     awsSqsSentencingOffenderEventsClient.purgeQueue(sentencingQueueOffenderEventsUrl).get()
     awsSqsSentencingOffenderEventsClient.purgeQueue(sentencingQueueOffenderEventsDlqUrl)?.get()
+    awsSqsSentencingMigrationClient.purgeQueue(sentencingMigrationUrl).get()
```

Comment:
not really required for this test but might as well leave it in since it is the right thing to do....